engine/src/main/java/org/pentaho/di/trans/steps/textfileoutput/TextFileOutput.java

```diff
@@ -1034,6 +1019,28 @@ private void createParentFolder( String filename ) throws Exception {
     }
   }
 
+  boolean isWriteEnclosureForFieldName( ValueMetaInterface v, String fieldName ) {
+    return ( isWriteEnclosed( v ) )
+            || ( !meta.isEnclosureFixDisabled() && containsSeparatorOrEnclosure( fieldName.getBytes(),
+            data.binarySeparator, data.binaryEnclosure ) );
+  }
+
+  boolean isWriteEnclosureForValueMetaInterface( ValueMetaInterface v ) {
+    return ( isWriteEnclosed( v ) )
+            || ( !meta.isEnclosureFixDisabled() && containsSeparatorOrEnclosure( v.getName().getBytes(),
+            data.binarySeparator, data.binaryEnclosure ) );
+  }
+
+  boolean isWriteEnclosureForWriteField( ValueMetaInterface v, byte[] str ) {
```

Comment:
you are getting a sonarQube code smell for the unused parameter v. https://sonar.orl.eng.hitachivantara.com/project/issues?id=org.pentaho.di%3Apdi&issues=AYUqAelXkhN_Ibp64Qlz&open=AYUqAelXkhN_Ibp64Qlz&pullRequest=8759